packages/dapp-high-roller/src/index.html

```diff
@@ -16,6 +16,26 @@
       href="https://fonts.googleapis.com/css?family=Dosis:500,700,800"
       rel="stylesheet"
     />
+    <!--
+      TODO: These builds are symlinked from node_modules and local packages.
+      THIS MUST NOT BE LIKE THIS! We must solve how Rollup is building
+      all of this.
+    -->
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/cuid/1.3.8/browser-cuid.js"></script>
+    <script src="/assets/eventemitter3.js"></script>
+    <script src="/assets/ethers.js"></script>
+
+    <script>
```

Comment:
```suggestion
    <script type="text/javascript">
```

Reply:
I wouldn't bother, Nothing of this should even be here.